apps/gov-website/pages/identity/connect.tsx

```diff
@@ -49,19 +42,30 @@ const Connect: NextPage<StaticProps> = ({
 	twitterRegistrarIndex,
 	discordRegistrarIndex,
 }) => {
-	const {
-		onSignInClick: onTwitterSignInClick,
-		username: twitterUsername,
-		clearUsername: clearTwitterUsername,
-	} = useSocialSignIn("Twitter");
-	const {
-		onSignInClick: onDiscordSignInClick,
-		username: discordUsername,
-		clearUsername: clearDiscordUsername,
-	} = useSocialSignIn("Discord");
+	return (
+		<Layout>
+			<Header />
+			<IdentityContext.Provider
+				twitterRegistrarIndex={twitterRegistrarIndex}
+				discordRegistrarIndex={discordRegistrarIndex}
+			>
+				<ConnectComponent />
```

Comment:
not sure there is a benefit of the `ConnectComponent` here

Reply:
I did this so I can move the `useSocialSignIn` params inside the provider and access them from the `connect` page directly, and also to avoid prop drilling of `twitterRegistrarIndex` & `discordRegistrarIndex`

Reply:
`ConnectPage`